utils/data.ts

```diff
@@ -2,6 +2,9 @@ import { EChartOption } from 'echarts';
 import gql from 'graphql-tag';
 import fetch from 'isomorphic-unfetch';
 import { PageScaffoldData } from '../components/DefaultLayout';
+import { DataSourcesLink } from '../components/DataSourcesSection';
+
+export const WebsiteUrl = 'http://devinit.org';
```

Comment:
What's this for?

Reply:
Its used to build the page url if an internal page is selected from the CMS instead of typing in a URL; I have now removed this constant and using `page.full_url` to return domain name as well as page path from the api